pkgs/servers/web-apps/discourse/rubyEnv/gemset.nix

```diff
@@ -1367,15 +1292,15 @@
     version = "2.5.8";
   };
   nokogiri = {
-    dependencies = ["mini_portile2" "racc"];
+    dependencies = ["racc"];
```

Comment:
@pdg137 the trick is to manually re-add `mini_portile2` to the `dependencies` section of `nokogiri`. No idea why this happens, but let's be honest: life's too short to debug rubygems issues :upside_down_face:

Reply:
Thanks! That worked great. There's still also the problem of it getting the wrong hash, but that's easy enough to fix.